### PR TITLE
Adds sitemap

### PR DESCRIPTION
**Pull request by AllenBW**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -115,5 +115,14 @@
   },
   "insights": {
     "appname": "advisor"
+  },
+  "routes": {
```
[the remark this answers is not in the document]
Comment (AllenBW) [reply package.json:119]: Routes.js doesn't quite capture the complete story though, its missing ah few that are listed elsewhere, this isn't a thing that changes too often, not worried about linkin em up...

MORE importantly... should we be including /insights/appname in the route? seems like we ought to.. but the rbac example doesn't... 😬

[the remark this answers is not in the document]
Comment (karelhala) [reply package.json:119]: There should be option for bundle/appName @PanSpagetka can give you details

[the remark this answers is not in the document]
Comment (PanSpagetka) [reply package.json:119]: Yes, there should be `prefix: insights` and `appname: advisor` as a part of the routes.

[the remark this answers is not in the document]
Comment (AllenBW) [reply package.json:119]: 👍

[the remark this answers is not in the document]
Comment (AllenBW) [reply package.json:119]: might need to trigger a release for https://github.com/RedHatInsights/frontend-components/pull/692

**Comment by AllenBW** [comment]
Looks like this lost favor n sitemaps don't matter anymore so closing it all

**Pull request closed without merging**